app/styles/_projects.less

```diff
@@ -72,9 +67,11 @@
       text-overflow: ellipsis;
     }
   }
-  p {
-    margin-bottom: 0;
-    margin-top: 5px;
+  small {
+    margin: 0;
```

Comment:
Nit:  [alphabetization of declarations by property](http://codeguide.patternfly.org/#css-declaration-order)